File: src/main/java/buttondevteam/discordplugin/util/Timings.java
package buttondevteam.discordplugin.util;

import buttondevteam.discordplugin.listeners.CommonListeners;

public class Timings {
	private long start;

	public Timings() {
		start = System.nanoTime();
	}

	public void printElapsed(String message) {
		CommonListeners.debug(message + " (" + (System.nanoTime() - start) / 1000000L + ")");
		start = System.nanoTime();
	}
}

File: src/main/java/buttondevteam/discordplugin/util/Timings.scala
package buttondevteam.discordplugin.util

import buttondevteam.discordplugin.listeners.CommonListeners

class Timings() {
    private var start = System.nanoTime

    def printElapsed(message: String): Unit = {
        CommonListeners.debug(message + " (" + (System.nanoTime - start) / 1000000L + ")")
        start = System.nanoTime
    }
}